projects/packages/phpunit-select-config/bin/phpunit-select-config.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * A shim to select a config file based on the PHPUnit version installed.
+ *
+ * @package automattic/phpunit-select-config
+ */
+
+// Make sure this script is being run over the PHP CLI.
+if ( 'cli' !== php_sapi_name() && 'phpdbg' !== php_sapi_name() ) {
+	return;
+}
```

Comment:
If we're going to be strict about this, perhaps we should throw an Exception for the benefit of whoever is running it?

Reply:
Yeah, that's probably a good idea. The main thing is that if somehow this manages to get into `vendor/` on a website that an attacker shouldn't be able to "run" it remotely.